releases/firmware-v9.2.1/assertion.txt

```diff
@@ -0,0 +1,8 @@
+By signing this file, the signer confirms that the firmware binary built from:
+
+git tag firmware/v9.2.1
+git commit hash e8de37ec83379c4ae55b346333a14fb98d4ebde4
+
+resulted, at the time of signing, in a firmware binary file with the following sha256sum:
+
+f31f0fcacadd2813413caf2f723c49c56a1a4d052aac3526f87e0b4141a01101
```

Comment:
@benma's and my build results in this hash:

    0100cc2fc39896738026a48c5030c0973852d1cdc173f14246cb156bb91db133

Are you sure you didn't have a dirty state? When you build, cmake outputs versions at the beginning. Check they don't contain `+dirty` flag.